Security/src/CloudFoundrySingleSignon/README-UAA.md

```diff
@@ -0,0 +1,122 @@
+﻿# CloudFoundry Single Signon Security Sample App
+
+ASP.NET Core sample app illustrating how to make use of the Steeltoe [CloudFoundry External Security Provider](https://docs.steeltoe.io/api/v3/security/) for Authentication and Authorization against a CloudFoundry OAuth2 security service (e.g. [UAA Server](https://github.com/cloudfoundry/uaa) or [Tanzu Single Signon Service](https://docs.pivotal.io/p-identity/)).
+
+> NOTE: For simplicity, we've moved the [instructions for utilizing the SSO Tile](README.md) with this sample
```

Comment:
Can you clarify this sentence? To/into/included/copied?

Reply:
I'm removing everything related to the direct UAA approach, I don't think that's a real use case